clients/cli/src/ui/dashboard.rs

```diff
@@ -261,95 +241,90 @@ pub fn render_dashboard(f: &mut Frame, state: &DashboardState) {
-    let status: List = {
-        let status_block = Block::default()
-            .borders(Borders::RIGHT)
-            .title("STATUS")
-            .style(
-                Style::default()
-                    .fg(Color::Cyan)
-                    .add_modifier(Modifier::BOLD),
-            );
+    // Status Section
+    let status_block = Block::default()
+        .borders(Borders::RIGHT)
+        .title("STATUS")
+        .style(
+            Style::default()
+                .fg(Color::Cyan)
+                .add_modifier(Modifier::BOLD),
+        );
 
-        let mut items: Vec<ListItem> = Vec::new();
+    let mut status_lines = Vec::new();
 
-        // Display the node ID, if any, or "Not connected" if not available
-        let node_id_text = if let Some(id) = state.node_id {
-            format!("NODE ID: {}", id)
-        } else {
-            "NODE ID: Not connected".to_string()
-        };
-        items.push(ListItem::new(node_id_text));
-
-        // Environment
-        items.push(ListItem::new(format!("ENVIRONMENT: {}", state.environment)));
-
-        // Version status
-        if state.update_available {
-            if let Some(latest) = &state.latest_version {
-                let version_text = format!("VERSION: {} → {} 🚀", version, latest);
-                items.push(ListItem::new(Line::from(vec![Span::styled(
-                    version_text,
-                    Style::default().fg(Color::LightYellow),
-                )])));
-            } else {
-                items.push(ListItem::new(Line::from(vec![Span::styled(
-                    "VERSION: Update Available 🚀",
-                    Style::default().fg(Color::LightYellow),
-                )])));
-            }
-        } else {
-            items.push(ListItem::new(format!("VERSION: {}", version)));
-        }
+    // Display the node ID, if any, or "Not connected" if not available
+    let node_id_text = if let Some(id) = state.node_id {
+        format!("NODE ID: {}", id)
+    } else {
+        "NODE ID: Not connected".to_string()
+    };
+    status_lines.push(Line::from(node_id_text));
 
-        // Uptime in Days, Hours, Minutes, Seconds
-        let uptime = state.start_time.elapsed();
-        let uptime_string = format!(
-            "UPTIME: {}d {}h {}m {}s",
-            uptime.as_secs() / 86400,
-            (uptime.as_secs() % 86400) / 3600,
-            (uptime.as_secs() % 3600) / 60,
-            uptime.as_secs() % 60
-        );
-        items.push(ListItem::new(uptime_string));
+    // Environment
+    status_lines.push(Line::from(format!("ENVIRONMENT: {}", state.environment)));
 
-        // NEX Points
-        if let Some(nex_points) = state.nex_points {
-            items.push(ListItem::new(format!("NEX POINTS: {}", nex_points)));
+    // Version status
+    if state.update_available {
+        if let Some(latest) = &state.latest_version {
+            let version_text = format!("VERSION: {} → {} 🚀", version, latest);
```

Comment:
I think we're generally trying to cut down on the emojis, so maybe remove here and below?